setup.py

```diff
@@ -8,6 +8,7 @@
 install_requires = [
     "requests",
     "posix_ipc",
+    "pyjwt",
```

Comment:
just to confirm: this ends up in the debian package dependencies right?

you'll probably also need to update `docs/requirements.txt` for doc building to work right still (shitty, sorry, i know :()